onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,26 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
-def aten_cross_entropy_loss(
-    self: TensorType,
-    target: TensorType,
-    weight: Optional[TensorType] = None,
-    reduction: int = 1,
-    ignore_index: INT64 = -100,
+@torch_op("aten::cross_entropy")
+def aten_cross_entropy(
+    self: TFloatOrBFloat16,
+    target: Sequence[int],
+    weight: Optional[TFloatOrBFloat16] = None,
+    reduction: str = "mean",
+    ignore_index: int = -100,
     label_smoothing: float = 0.0,
-) -> TensorType:
+) -> TFloatOrBFloat16:
     # cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, SymInt ignore_index=-100, float label_smoothing=0.0) -> Tensor
 
-    raise NotImplementedError()
+    if op.OptionalHasElement(weight):
+        result, _ = op.SoftmaxCrossEntropyLoss(
```

Comment:
```suggestion
        result = op.SoftmaxCrossEntropyLoss(
```

I think we can just do this as ONNX knows to output just one thing when there is only one output connected